packages/blocks/src/divider-block/divider-model.ts

```diff
@@ -2,14 +2,15 @@ import { Page, BaseBlockModel, IBaseBlockProps } from '@blocksuite/store';
 
 export interface DividerBlockProps extends IBaseBlockProps {
   flavour: 'affine:divider';
+  type: 'divider';
```

Comment:
Why do we need this `type` field? If fixing the image block requires all other blocks to define a different `type` field, then the way of the fix seems to be problematic.